backend/wayland/backend.c

```diff
@@ -59,6 +60,25 @@ static const struct xdg_wm_base_listener xdg_wm_base_listener = {
 	xdg_wm_base_handle_ping,
 };
 
+static void linux_dmabuf_v1_handle_format(void *data,
+		struct zwp_linux_dmabuf_v1 *linux_dmabuf_v1, uint32_t format) {
+	// This event is deprecated, we use the modifier event instead
```

Comment:
Do we know if all compositors always use the modifier event if we bind the latest version? E.g. will someone keep using format on AMDGPU without modifiers?
I think it would be worth just implementing this anyway, since it's just a single call to `wlr_drm_format_set_add(&formats, format, DRM_FORMAT_MOD_INVALID)`.

Reply:
>will someone keep using format on AMDGPU without modifiers?

Yeah, the compositor should send a `modifier` event with `DRM_FORMAT_MOD_INVALID`.

I still added the call as suggested, just in case.